container/lxd/container.go

```diff
@@ -181,8 +202,11 @@ func (s *Server) ContainerAddresses(name string) ([]network.Address, error) {
 // If the container fails to be started, it is removed.
 // Upon successful creation and start, the container is returned.
 func (s *Server) CreateContainerFromSpec(spec ContainerSpec) (*Container, error) {
+	logger.Infof("starting container %q (image %q)...", spec.Name, spec.Image.Image.Filename)
```

Comment:
Are we going to put any additional logging here like with 2.5?
- https://bugs.launchpad.net/juju/+bug/1779897

Reply:
Yeah, I'll remember to make that change here for later forward-porting.